src/mapping/recurrences/attractor_mapping_recurrences.jl

```diff
@@ -0,0 +1,326 @@
+  The keyword `Dt` can also be used instead if `Δ` (`\\Delta`) is not accessible.
+  It is `1` for discrete time systems.
+  For continuous systems, an automatic value is calculated using
+  [`automatic_Δt_basins`](@ref). For very fine grids, this can become very small,
+  much smaller than the typical integrator internal step size in case of adaptive
+  integrators. In such cases, use `force_non_adaptive = true`.
+* `force_non_adaptive = false`: Only used if the input dynamical system is `CoupledODEs`.
+  If `true` the additional keywords `adaptive = false, dt = Δt` are given as `diffeq`
+  to the `CoupledODEs`. This means that adaptive integration is turned off and `Δt` is
+  used as the ODE integrator timestep. This is useful in (1) very fine grids, and (2)
+  if some of the attractors are limit cycles. We have noticed that in this case the
+  integrator timestep becomes commensurate with the limit cycle period, leading to
+  incorrectly counting the limit cycle as more than one attractor.
+
+### Finite state machine configuration
+
+* `mx_chk_att = 2`: Μaximum checks of consecutives hits of an existing attractor cell
+  before declaring convergence to that existing attractor.
+* `mx_chk_hit_bas = 10`: Maximum check of consecutive visits of the same basin of
+  attraction before declaring convergence to an existing attractor.
+  This is ignored if `sparse = true`, as basins are not stored internally.
+* `mx_chk_fnd_att = 100`: Maximum check of consecutive visits to a previously visited
+  unlabeled cell before declaring we have found a new attractor.
+* `mx_chk_loc_att = 100`: Maximum check of consecutive visits to cells marked as a new
+  attractor, during the attractor identification phase, before declaring we that we have
+  identified the new attractor with sufficient accuracy.
+* `store_once_per_cell = true`: Control if multiple points in state space that belong to
+  the same cell are stored or not in the attractor, after an attractor is found.
+  If `true`, each visited cell will only store a point once, which is desirable for fixed
+  points and limit cycles. If `false`, at least `mx_chk_loc_att` points are
+  stored per attractor, leading to more densely stored attractors,
+  which may be desirable for instance in chaotic attractors.
+* `mx_chk_lost = 20`: Maximum check of iterations outside the defined grid before we
+  declare the orbit lost outside and hence assign it label `-1`.
+* `horizon_limit = 1e6`: If the norm of the integrator state reaches this
+  limit we declare that the orbit diverged to infinity.
+* `mx_chk_safety = Int(1e6)`: A safety counter that is always increasing for
+  each initial condition. Once exceeded, the algorithm assigns `-1` and throws a warning.
+  This clause exists to stop the algorithm never haulting for innappropriate grids,
+  where a found attractor may intersect in the same cell with a new attractor the orbit
+  traces (which leads to infinite resetting of all counters).
```

Comment:
```suggestion
  This clause exists to stop the algorithm never halting for inappropriate grids. It may happen 
  when a newly found attractor orbit intersects in the same cell of a previously found attractor (which leads to infinite resetting of all counters).
```